project/GrasshopperBuild.scala

```diff
@@ -141,6 +142,7 @@ object GrasshopperBuild extends Build {
         assemblyJarName in assembly := {s"grasshopper-${name.value}.jar"},
         assemblyMergeStrategy in assembly := {
           case "application.conf" => MergeStrategy.concat
+          case PathList("org", "joda", "time", "base", "BaseDateTime.class") => MergeStrategy.first
           case x =>
```

Comment:
Please add a comment here as to why we're doing this, with links to any relevant resources.